package.json

```diff
@@ -18,6 +18,9 @@
     "build/index.js",
     "build/example.js"
   ],
+  "dependencies": {
+      "prop-types": "^15.5.10"
```

Comment:
One option might be to tidy up this branch, then publish an 2.0 alpha version in npm under the next tag. So you'd be able to install this newer version with `npm install gridicons@next`